src/main/java/org/powerflows/dmn/io/yaml/CustomPropertyUtils.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2018-present PowerFlows.org - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.powerflows.dmn.io.yaml;
+
+import org.yaml.snakeyaml.error.YAMLException;
+import org.yaml.snakeyaml.introspector.BeanAccess;
+import org.yaml.snakeyaml.introspector.MissingProperty;
+import org.yaml.snakeyaml.introspector.Property;
+import org.yaml.snakeyaml.introspector.PropertyUtils;
+
+import java.util.Map;
+
+public class CustomPropertyUtils extends PropertyUtils {
+    @Override
+    public Property getProperty(Class<? extends Object> type, String name, BeanAccess bAccess) {
+        Map<String, Property> properties = getPropertiesMap(type, bAccess);
+        Property property = properties.get(name);
+        if (property == null && name.contains("-")) {
+            final String camelName = toCamelCase(name);
+            property = properties.get(camelName);
+            if (property != null) {
+                properties.put(name, property);
+            }
+        }
+        if (property == null && isSkipMissingProperties()) {
+            property = new MissingProperty(name);
+        }
+        if (property == null) {
+            throw new YAMLException(
+                    "Unable to find property '" + name + "' on class: " + type.getName());
```

Comment:
Is this wrapping needed? The line looks pretty short.

Reply:
as above